db/db_impl/db_impl_open.cc

```diff
@@ -233,6 +242,10 @@ Status DBImpl::ValidateOptions(const DBOptions& db_options) {
         "atomic_flush is incompatible with enable_pipelined_write");
   }
 
+  if (db_options.file_system != nullptr && !db_options.env) {
```

Comment:
The default value of `options.env` is `Env::Default()`. Therefore, only when the application explicitly sets it to null can this happen. Is this correct?
In this case, will it work if we set it to `Env::Default()`?

Reply:
Yes, we can do that.